var/spack/repos/builtin/packages/superlu/package.py

```diff
@@ -181,9 +184,17 @@ def test(self):
             args.append('HEADER=' + self.prefix.include)
         args.append('superlu')
 
-        test_dir = join_path(self.install_test_root, self.examples_src_dir)
+        test_dir = join_path(self.test_suite.current_test_cache_dir,
+                             self.examples_src_dir)
+
+        if not os.path.exists(test_dir):
+            print('Skipping superlu test')
```

Comment:
This needs a more informative message, perhaps:

```suggestion
            print('Skipping superlu test: missing required {0}'.format(test_dir))
```